front/src/reducers/editor/__tests__/editorReducer.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { EditorState } from 'applications/editor/tools/types';
+import { createStoreWithoutMiddleware } from 'Store';
+import editorTestDataBuilder from './editorTestDataBuilder';
+import {
+  editorInitialState,
+  selectLayers,
+  editorSlice,
+  loadDataModelAction,
+  updateTotalsIssueAction,
+  updateFiltersIssueAction,
+} from '..';
+
+const createStore = (initialStateExtra?: EditorState) =>
+  createStoreWithoutMiddleware({
+    [editorSlice.name]: initialStateExtra,
+  });
+let store: ReturnType<typeof createStore>;
+
+beforeEach(() => {
+  store = createStore();
+});
+
+describe('editorReducer', () => {
+  const testDataBuilder = editorTestDataBuilder();
+  it('should return initial state', () => {
+    const editorState = store.getState()[editorSlice.name];
+    expect(editorState).toEqual(editorInitialState);
+  });
+
+  it('should handle selectLayers', () => {
+    const editorLayers = testDataBuilder.buildEditorLayers(['catenaries', 'routes']);
+    store.dispatch(selectLayers(editorLayers));
+    const editorState = store.getState()[editorSlice.name];
+    expect(editorState.editorLayers).toEqual(new Set(['catenaries', 'routes']));
+  });
+
+  describe('should handle loadDataModelAction', () => {
+    it('should update editorSchema', () => {
```

Comment:
You can even merge the describe and the it here
```suggestion
  it('should handle loadDataModelAction and update editorSchema', () => {
```

Reply:
Done